### PR TITLE
BUG/TST: Improved Compatibility with pysat and Travis CI

### PR DESCRIPTION
**Pull request by jklenzing**
Addresses #16.  Numerous updates, including
- Updates to pysat syntax compatible with version 2.0.0
- Update to Travis CI xenial environment
  - Added tests for python 3.7
  - Use of `no_sgp4` branch of `pysat` to streamline
- Updates to f2py usage in setup
- Fix links in documentation to new pysat organization
- Spelling and Whitespace

### CODE REVIEW

**Comment by jklenzing** [comment]
Updates to f2py were based on apexpy during debugging.  Potentially, these are not needed.

File: pysatMagVect/tests/test_core.py [hunk 1]
```diff
@@ -556,7 +556,7 @@ def test_unit_vector_plots(self):
                 self.inst[:,'sc_xhat_x'], self.inst[:,'sc_xhat_y'], self.inst[:,'sc_xhat_z'] = 1., 0., 0.
                 self.inst[:,'sc_yhat_x'], self.inst[:,'sc_yhat_y'], self.inst[:,'sc_yhat_z'] = 0., 1., 0.
                 self.inst[:,'sc_zhat_x'], self.inst[:,'sc_zhat_y'], self.inst[:,'sc_zhat_z'] = 0., 0., 1.
-                self.inst.data.index = pysat.utils.season_date_range(pysat.datetime(2000,1,1),
+                self.inst.data.index = pysat.utils.time.create_date_range(pysat.datetime(2000,1,1),
```
Comment (rstoneback) [inline pysatMagVect/tests/test_core.py:559]: I think we need to be more careful changing method names in pysat in the future. The public version of pysat doesn't have this call. Perhaps we should have a method with the old name, for at least a version, that raises a Deprecation Warning and transparently calls the new routine. Technically, we can't release this code for use until the next version of pysat.

**Review by rstoneback — APPROVED** [review]
Thanks Jeff.

I'm going to create an issue to remove the dependance on the no_sgp4 branch once we get pysat sorted out.

File: pysatMagVect/tests/test_core.py (hunk 1 above)
Comment (jklenzing) [reply pysatMagVect/tests/test_core.py:559]: Good catch.

**Comment by jklenzing** [comment]
The no_sgp4 is a temporary fix.  We should chat at the next meeting.  I think I have this on the agenda, but I'll double-check.  We could add a function with the old name (season_date_range) that points to the new name (create_date_range) with a deprecation warning to maintain compatibility for now.

**Pull request merged**